src/nfts/NftGallery.tsx

```diff
@@ -0,0 +1,132 @@
+import { useDispatch, useSelector } from 'react-redux'
+import { NftEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Touchable from 'src/components/Touchable'
+import ImageErrorIcon from 'src/icons/ImageErrorIcon'
+import DrawerTopBar from 'src/navigator/DrawerTopBar'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import NftImage from 'src/nfts/NftImage'
+import NftsLoadError from 'src/nfts/NftsLoadError'
+import { nftsErrorSelector, nftsLoadingSelector, nftsSelector } from 'src/nfts/selectors'
+import { fetchNfts } from 'src/nfts/slice'
+import { NftOrigin } from 'src/nfts/types'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+
+// This is used instead of a gap between the images because the gap is not supported in RN yet.
+const imageSize = (variables.width - Spacing.Regular16 * 3) / 2
+
+export default function NftGallery() {
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+  const loading = useSelector(nftsLoadingSelector)
+  const error = useSelector(nftsErrorSelector)
+  const nfts = useSelector(nftsSelector)
+
+  useEffect(() => {
+    ValoraAnalytics.track(NftEvents.nft_gallery_screen_open)
+    dispatch(fetchNfts())
+  }, [])
+
+  return (
+    <SafeAreaView testID="NftGallery" style={styles.safeAreaContainer} edges={['top']}>
+      <DrawerTopBar middleElement={<Text>{t('nftGallery.title')}</Text>} />
+      {error ? (
+        <NftsLoadError testID="NftGallery/NftsLoadErrorScreen" />
+      ) : (
+        <FlatList
+          numColumns={2}
+          data={nfts}
+          refreshing={loading}
+          onRefresh={() => dispatch(fetchNfts())}
+          contentContainerStyle={styles.contentContainer}
+          refreshControl={
+            <RefreshControl
+              refreshing={loading}
+              colors={[colors.greenUI]}
+              onRefresh={() => dispatch(fetchNfts())}
+            />
+          }
+          renderItem={({ item, index }) => (
+            <Touchable
+              borderless={false}
+              onPress={() => navigate(Screens.NftsInfoCarousel, { nfts: [item] })}
+              style={[
+                styles.touchableIcon,
+                index % 2 === 0
+                  ? { marginRight: Spacing.Regular16 } &&
+                    index === nfts.length - 1 &&
+                    styles.lastImage
+                  : { marginLeft: Spacing.Regular16 },
+              ]}
+            >
+              <NftImage
+                nft={item}
+                testID="NftGallery/NftImage"
+                width={imageSize}
+                height={imageSize}
+                ErrorComponent={
+                  <View style={styles.errorView}>
+                    <ImageErrorIcon color="#C93717" />
```

Comment:
can we use the red from the colors constant?

Reply:
We use `#C93717` in `NftsInfoCarousel.tsx` also. I think it would be a bit strange if we displayed different colors of red. We could swap both, add this new color as a constant or keep it the way it is.